dataservice/api/genomic_file/schemas.py

```diff
@@ -9,9 +9,12 @@
 
 DATA_TYPE_ENUM = {'Aligned Reads',
                   'Aligned Reads Index',
+                  'Unaligned Reads',
                   'Simple Nucleotide Variation',
                   'Variant Calls',
-                  'Individual Variant Calls',
```

Comment:
are we re-mapping `Individual Variant Calls` to `Variant Calls` ? bc i see 2186 genomic files with data_type `Individual Variant Calls`.

Reply:
yep